lib/core/mount.js

```diff
@@ -146,10 +146,11 @@ function mount(mountPath, parentApp, events) {
 					process.exit(1);
 				}
 				break;
-			
+
 			case 'redis':
 				// default session store for using Redis
-				sessionStore = 'connect-redis';
+			  sessionStore = 'connect-redis';
+
 				// TODO: are there default options that should be applied here?
```

Comment:
could you look into whether or not we should pass default options here? if not please also remove both TODOs